dev-packages/ext-scripts/package.json

```diff
@@ -13,11 +13,11 @@
     "ext:build": "concurrently -n compile,lint -c blue,green \"theiaext compile\" \"theiaext lint\"",
     "ext:compile": "tsc -b compile.tsconfig.json",
     "ext:compile:clean": "rimraf lib *.tsbuildinfo",
-    "ext:lint": "eslint --cache=true --no-error-on-unmatched-pattern=true \"{src,test}/**/*.{ts,tsx}\"",
+    "ext:lint": "if-env SKIP_LINT=true && echo 'skip lint check' || eslint --cache=true --no-error-on-unmatched-pattern=true \"{src,test}/**/*.{ts,tsx}\"",
     "ext:lint:clean": "rimraf .eslintcache",
     "ext:watch": "tsc -p compile.tsconfig.json -w",
     "ext:watch:slow": "tsc -b compile.tsconfig.json -w",
-    "ext:test": "nyc mocha --opts ../../configs/mocha.opts \"./lib/**/*.*spec.js\"",
+    "ext:test": "if-env SKIP_TEST=true && echo 'skip test' || nyc mocha --opts ../../configs/mocha.opts \"./lib/**/*.*spec.js\"",
```

Comment:
Tests are not ran as part of `yarn install` right? Is this meant to speed up a specific yarn script?

Reply:
yes tests are not executed by `yarn install`

I've applied SKIP pattern to have like a way to disable it if someone enable tests somewhere but if you think it's useless I can remove for test as it's not enabled by default (or we enable tests by default and keep SKIP but it's a different story)

Reply:
> or we enable tests by default and keep SKIP but it's a different story

If anything I'd trim down what `prepare` is doing rather than adding more to it... But it is a different story indeed.

If neither of us see how to use this flag then let's get rid of it indeed, we can always add it when we need it.

Reply:
I've removed the SKIP_TEST